src/posts/2020-09-11-announcing-alchemy-conf.md

```diff
@@ -0,0 +1,118 @@
+since, at the time, when we stared at our bank account statement, we saw 200€
+staring back at us. The remaining 50€ were put out of our own pockets. I can
+never give a correct answer to the question of "how important the community is,”
+but **I can look back and proudly say it has always been a part of us.**
+
+A lot more followed. **We started organizing several meetups** (Braga.rb,
+Braga.js, Braga.Design, Braga.Product, Braga BEAM), **helped others by hosting
+them in our office, eventually went around to organize [Ruby Conf
+Portugal][rubyconf] from 2014 to 2016. From 2016 to 2018, we also had [Mirror
+Conf][mirrorconf]**, a conference that aimed to become a conversation between
+designers and front-end developers. Doing this for years on end took its toll,
+of course, and we needed a break.
+
+We have evolved from working mainly as a Ruby on Rails consultancy firm to a
+venture studio during the past few years, where we help create new startups.
+With this transition, we found Elixir to be a valuable alternative that we
+should invest, and we’ve been working with it for more than four years. Many of
+us have been speakers at conferences, written blog posts, talked in podcasts and
+screencasts, etc. **We still look to inspire those around us, and we can sense a
+real vibrancy in the booming Elixir community in Portugal.** It feels like 2014
+again, with the Ruby community. **It’s contagious, inspiring, and it urges us to
+jump into action.** That’s why we’re back organizing conferences. Big time.
+
+## What it is
+
+**Alchemy Conf will be a two-day Elixir conference held in the lovely city of
+Braga, Portugal.** Braga is Portugal’s third-largest city. It’s beautiful,
+history-rich, with a fantastic university, filled with young potential. We have
+a booming tech ecosystem here, encompassing fully fledged international
+companies and promising startups.
+
+**The conference will take place on the 14th and 15th of May 2021. It will be a
+single-track conference, with a mixture of guest speakers and CFP selections.**
+The first speaker announcements are just right around the corner, and the CFPs
+will open soon.
+
+**[The early birds are on sale now][tickets].** We are looking into student
+discounts, so if you are a student, send us an email to [hello@alchemyconf.com].
+
+Like the previous conferences we organized, Alchemy Conf is community-driven.
+This means it is a non-profit. Our focus is and always has been to inspire the
+community. To create an opportunity for developers and enthusiasts to get
+together, to share and learn, experience, and network. From the attendees to the
+speakers, we want everybody to feel like Alchemy Conf is a little bit of their
+own. This is why we want to fully commit to transparency. **We will be making a
+transparency report containing the expenses and earnings of the conference. Any
+profit that may arise will be split equally amongst the speakers.**
```

Comment:
```suggestion
profit that may arise will be given back to the community.**
```

This gives us more wiggle room, keeping the assurance that it's a not for profit event.

Reply:
Adding your suggestion, but I'd be more comfortable if we were upfront with what we are doing. I think we should finish the discussion on this topic on Basecamp, before the announcement is published. To make sure everyone is on board.